### PR TITLE
Add tests for Util

### PR DESCRIPTION
**Pull request by sharidas**
Add tests for Util

Signed-off-by: Sujith H <sharidasan@owncloud.com>

### CODE REVIEW

File: tests/lib/UtilTest.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+			['admin']
+			];
+	}
+
+	/**
+	 * @dataProvider impersonator
+	 * @param $impersonator
+	 */
+	public function testSwitchUser($impersonator) {
+		$user = $this->createMock('\OCP\IUser');
```
Comment (DeepDiver1975) [inline tests/lib/UtilTest.php]: IUser::class

File: tests/lib/UtilTest.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ *
+ */
+
+namespace OCA\Impersonate\Tests\Lib;
+
+use OCA\Impersonate\Util;
+use Test\TestCase;
+
+class UtilTest extends TestCase {
+	private $session;
```
Comment (DeepDiver1975) [inline tests/lib/UtilTest.php]: add PHPDoc to fields

File: tests/lib/UtilTest.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	private $tokenProvider;
+	private $userSession;
+	private $request;
+	private $util;
+	public function setUp() {
+
+		$this->session = $this->createMock(\OCP\ISession::class);
```
Comment (DeepDiver1975) [inline tests/lib/UtilTest.php]: use use declaration

**Comment by sharidas** [comment]
@DeepDiver1975 Made changes as requested.

**Pull request merged**